package.json

```diff
@@ -14,25 +14,25 @@
   },
   "dependencies": {
     "@babel/compat-data": "^7.11.0",
-    "@bcrs-shared-components/action-chip": "1.0.6",
-    "@bcrs-shared-components/breadcrumb": "2.0.1",
-    "@bcrs-shared-components/business-lookup": "1.0.2",
-    "@bcrs-shared-components/certify": "2.0.4",
-    "@bcrs-shared-components/completing-party": "2.0.6",
-    "@bcrs-shared-components/confirm-dialog": "1.1.0",
-    "@bcrs-shared-components/contact-info": "1.1.2",
-    "@bcrs-shared-components/corp-type-module": "1.0.6",
-    "@bcrs-shared-components/court-order-poa": "1.0.11",
-    "@bcrs-shared-components/date-picker": "1.1.9",
-    "@bcrs-shared-components/detail-comment": "1.0.4",
-    "@bcrs-shared-components/enums": "1.0.19",
-    "@bcrs-shared-components/fee-summary": "1.1.15",
-    "@bcrs-shared-components/folio-number": "1.0.0",
+    "@bcrs-shared-components/action-chip": "1.0.27",
+    "@bcrs-shared-components/breadcrumb": "2.0.5",
+    "@bcrs-shared-components/business-lookup": "1.0.3",
+    "@bcrs-shared-components/certify": "2.0.8",
+    "@bcrs-shared-components/completing-party": "2.0.7",
+    "@bcrs-shared-components/confirm-dialog": "1.1.1",
+    "@bcrs-shared-components/contact-info": "1.1.9",
+    "@bcrs-shared-components/corp-type-module": "1.0.7",
+    "@bcrs-shared-components/court-order-poa": "2.0.8",
+    "@bcrs-shared-components/date-picker": "1.1.19",
+    "@bcrs-shared-components/detail-comment": "1.0.13",
+    "@bcrs-shared-components/enums": "1.0.20",
+    "@bcrs-shared-components/fee-summary": "1.1.22",
+    "@bcrs-shared-components/folio-number": "1.0.3",
     "@bcrs-shared-components/help-business-number": "1.0.0",
-    "@bcrs-shared-components/interfaces": "1.0.39",
-    "@bcrs-shared-components/nature-of-business": "1.0.8",
-    "@bcrs-shared-components/share-structure": "2.0.0",
-    "@bcrs-shared-components/staff-payment": "1.0.35",
+    "@bcrs-shared-components/interfaces": "1.0.40",
+    "@bcrs-shared-components/nature-of-business": "1.0.10",
+    "@bcrs-shared-components/share-structure": "2.0.1",
+    "@bcrs-shared-components/staff-payment": "2.0.10",
```

Comment:
Did you test all these?

(I know that some of these are layout changes.)

If it was me, I'd only change the ones I needed (and I'd look at their history and test accordingly).

Reply:
I tested the Major's, the patch versions - I left mostly

Reply:
Please add a comment to your ticket for QA to pay special attention to the shared components you updated.

Reply:
Done, should be in the ticket.